app/helpers/hyrax/batch_select_helper.rb

```diff
@@ -0,0 +1,11 @@
+# View Helpers for Hydra Batch Edit functionality
+module Hyrax
+  module BatchSelectHelper
+    # Displays the button to select/deselect items for your batch.  Call this in the index
+    # partial that's rendered for each search result.
+    # @param [Hash] document the Hash (aka Solr hit) for one Solr document
+    def button_for_add_to_batch(document)
```

Comment:
Discovered that using `render` in some views works and I _guess_ the render call is now _sorta_ using the controller module, which now includes this module.  For some reason, the views were not _finding_ this module/method automatically and I noticed that the batch edit modules have some explicit helper includes, so I followed that pattern for the batch select.